### ISSUE TITLE
Server Crashes on startup after 1.0.1 update

### ISSUE THREAD

**Issue by clbx** [TeamAOF/All-of-Fabric-7#64]
#57 fixed the previous server crash on startup running 1.0.0. Running the 1.0.1 update I get this:

```
[17:29:16] [Server thread/ERROR]: Encountered an unexpected exception
java.util.ConcurrentModificationException: null
	at java.util.HashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.class_26.method_125(class_26.java:120) ~[server-intermediary.jar:?]
	at wraith.fwaystones.util.WaystoneStorage.saveWaystones(WaystoneStorage.java:177) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
	at wraith.fwaystones.util.WaystoneStorage.tryAddWaystone(WaystoneStorage.java:151) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
	at wraith.fwaystones.util.WaystoneStorage$Lazy.getEntity(WaystoneStorage.java:352) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
	at wraith.fwaystones.util.WaystoneStorage.toTag(WaystoneStorage.java:118) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
	at wraith.fwaystones.util.WaystoneStorage$1.method_75(WaystoneStorage.java:52) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
	at net.minecraft.class_18.method_17919(class_18.java:36) ~[server-intermediary.jar:?]
	at net.minecraft.class_26.method_32384(class_26.java:122) ~[server-intermediary.jar:?]
	at java.util.HashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.class_26.method_125(class_26.java:120) ~[server-intermediary.jar:?]
	at wraith.fwaystones.util.WaystoneStorage.saveWaystones(WaystoneStorage.java:177) ~[wraith-waystones-3.3.1+mc1.20.1.jar:?]
```

Same setup, server is running on the itzg minecraft docker container with Java 17.0.9

Full Log: [minecraft-modded.log](https://github.com/TeamAOF/All-of-Fabric-7/files/13628685/minecraft-modded.log)

**Comment by ImAK9** [comment]
Reset/delete the data folders of waystones

**Comment by clbx** [comment]
Where is that directory

**Comment by ImAK9** [comment]
Should be in your world folder

**Comment by clbx** [comment]
``/world/data/fw_waystones.dat`` ?

**Comment by ImAK9** [comment]
Yup, that's the one

**Comment by clbx** [comment]
That worked, thank you!

**Comment by clbx** [comment]
Every time the server starts up now, it crashes on the waystones.

**Comment by Kazuhiko-Gushiken** [comment]
@clbx we know. we are currently communicating with the Fabric Waystones developer to find a fix as we speak

**Comment by ImAK9** [comment]
Fixed in 1.0.2
